audit/audit.go

```diff
@@ -0,0 +1,136 @@
+package audit
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os/user"
+
+	_ "github.com/lib/pq"
+	"github.com/sirupsen/logrus"
+	"go.mozilla.org/sops/logging"
+	"gopkg.in/yaml.v2"
+)
+
+var log *logrus.Logger
+
+func init() {
+	log = logging.NewLogger("AUDIT")
+	confBytes, err := ioutil.ReadFile(configFile)
+	if err != nil {
+		log.WithField("error", err).Debugf("Error reading config")
+		return
+	}
+	var conf config
+	err = yaml.Unmarshal(confBytes, &conf)
+	if err != nil {
+		log.WithField("error", err).Panicf("Error unmarshalling config")
+	}
+	// If we are running test, then don't create auditors.
+	// This is pretty hacky, but doing it The Right Way would require
+	// restructuring SOPS to use dependency injection instead of just using
+	// globals everywhere.
+	if flag.Lookup("test.v") != nil {
+		return
+	}
+	for _, pgConf := range conf.Backends.Postgres {
+		auditors = append(auditors, NewPostgresAuditor(pgConf.ConnStr))
+	}
+}
+
+// TODO: Make platform agnostic
+const configFile = "/etc/sops/audit.yaml"
```

Comment:
that's actually pretty easy to do: just move this line into a file called `auditenv_posix.go`, and create one called `auditenv_windows.go` that sets the location to a windows-friendly path. Not sure if it's required for this iteration of the feature though.